plusplusbot/tests/fixtures/history.json

```diff
@@ -0,0 +1 @@
mestamp": 1592265600.0, "user_id": "U5HKU1Q0W"}, {"operation": "++", "timestamp": 1592269200.0, "user_id": "U985L6R1M"}, {"operation": "++", "timestamp": 1592272800.0, "user_id": "U985L6R1M"}, {"operation": "++", "timestamp": 1592276400.0, "user_id": "U0VCW825A"}, {"operation": "++", "timestamp": 1592280000.0, "user_id": "U0ZC11HC7"}, {"operation": "++", "timestamp": 1592283600.0, "user_id": "U0ZC11HC7"}, {"operation": "++", "timestamp": 1592287200.0, "user_id": "U0ZC11HC7"}, {"operation": "++", "timestamp": 1592290800.0, "user_id": "U985L6R1M"}, {"operation": "++", "timestamp": 1592348400.0, "user_id": "U0VCW825A"}, {"operation": "++", "timestamp": 1592352000.0, "user_id": "U0ZC11HC7"}, {"operation": "++", "timestamp": 1592355600.0, "user_id": "U985L6R1M"}, {"operation": "++", "timestamp": 1592359200.0, "user_id": "U5HKU1Q0W"}, {"operation": "++", "timestamp": 1592362800.0, "user_id": "U0VCW825A"}, {"operation": "++", "timestamp": 1592434800.0, "user_id": "U0ZC11HC7"}, {"operation": "++", "timestamp": 1592438400.0, "user_id": "U5HKU1Q0W"}, {"operation": "++", "timestamp": 1592442000.0, "user_id": "U0VCW825A"}, {"operation": "++", "timestamp": 1592445600.0, "user_id": "U5HKU1Q0W"}, {"operation": "++", "timestamp": 1592449200.0, "user_id": "U0VCW825A"}, {"operation": "++", "timestamp": 1592452800.0, "user_id": "U985L6R1M"}, {"operation": "++", "timestamp": 1592456400.0, "user_id": "U0VCW825A"}, {"operation": "++", "timestamp": 1592460000.0, "user_id": "U985L6R1M"}, {"operation": "++", "timestamp": 1592463600.0, "user_id": "U985L6R1M"}, {"operation": "++", "timestamp": 1592521200.0, "user_id": "U985L6R1M"}, {"operation": "++", "timestamp": 1592524800.0, "user_id": "U985L6R1M"}, {"operation": "++", "timestamp": 1592528400.0, "user_id": "U0ZC11HC7"}, {"operation": "++", "timestamp": 1592532000.0, "user_id": "U985L6R1M"}, {"operation": "++", "timestamp": 1592535600.0, "user_id": "U0ZC11HC7"}, {"operation": "--", "timestamp": 1592539200.0, "user_id": "U0ZC11HC7"}, {"operation": "++", "timestamp": 1592542800.0, "user_id": "U5HKU1Q0W"}, {"operation": "++", "timestamp": 1592546400.0, "user_id": "U5HKU1Q0W"}, {"operation": "++", "timestamp": 1592550000.0, "user_id": "U0ZC11HC7"}, {"operation": "++", "timestamp": 1592553600.0, "user_id": "U5HKU1Q0W"}, {"operation": "++", "timestamp": 1592607600.0, "user_id": "U5HKU1Q0W"}, {"operation": "++", "timestamp": 1592611200.0, "user_id": "U0VCW825A"}, {"operation": "++", "timestamp": 1592614800.0, "user_id": "U985L6R1M"}, {"operation": "++", "timestamp": 1592618400.0, "user_id": "U0VCW825A"}, {"operation": "++", "timestamp": 1592622000.0, "user_id": "U985L6R1M"}, {"operation": "++", "timestamp": 1592625600.0, "user_id": "U0VCW825A"}, {"operation": "++", "timestamp": 1592629200.0, "user_id": "U0VCW825A"}, {"operation": "++", "timestamp": 1592632800.0, "user_id": "U985L6R1M"}, {"operation": "++", "timestamp": 1592636400.0, "user_id": "U985L6R1M"}, {"operation": "++", "timestamp": 1592640000.0, "user_id": "U5HKU1Q0W"}]
```

Comment:
Could we pretty print this? Will make it easier later to modify if we need to, and won't really result in any bad things?